### ISSUE TITLE
Ghosting someone into a cult juggernaut does not make them a cultist and you cannot set them as such.

### ISSUE THREAD

**Issue by morrowwolf** [yogstation13/yogstation#2824]
[Note]: Ghosting someone into a cult juggernaut does not make them a cultist and you cannot set them as one in the traitor panel.  Dunno about the others shade etc.

[Admins]: I ghosted a dude in.

**Comment by Ratvar** [comment]
You sure that juggernauth was cult originally? Constructs made out of ghosted fuckdudes are noncult, suppose jug was noncult, tis why.

**Comment by ShadowDeath6** [comment]
constructs by default are noncult because of a pr that added factions to constructs iirc

**Comment by Time-Green** [comment]
I mace the construct “factions” and this shouldn’t be happening

**Comment by morrowwolf** [comment]
Yeah I'm going off them not showing up as cult in the antaghud when the other juggernaut did and not having faction set to anything and not being able to see other cultists when the other juggernaut could.  Honestly haven't tried to replicate it but I've seen it twice now when ghosting people into constructs like shades, juggernaut, etc.

**Comment by morrowwolf** [comment]
I should also clarify I am using the option in view variables "offer to ghosts".  I am not just changing the ckey to someone else's.  Unsure if it matters.

**Comment by Time-Green** [comment]
It doesn’t. And are you sure it wasn’t just a broken hud?

**Comment by morrowwolf** [comment]
I turned the antaghud off and on as it does break every now and then.  The main reason I've made this issue is because the two juggernauts had different variables regarding faction/hud/traitor panel etc.  If I see this again I'll be sure to get the exact differences between the two.
